spec/graphql/execution/errors_spec.rb

```diff
@@ -3,6 +3,14 @@
 
 describe "GraphQL::Execution::Errors" do
   class ErrorsTestSchema < GraphQL::Schema
+    def self.object_from_id(id, ctx)
+      if id == 1
+        :thing
+      else
+        raise ErrorD
+      end
+    end
```

Comment:
Is this addition still used, or should it be removed? (I don't quite see how it's used in the tests 🙈 )

Reply:
@rmosolgo Thank you again!

spec/graphql/execution/errors_spec.rb:177

`describe "errors raised in input_object loads` need this.

Without this, `GraphQL::RequiredImplementationMissingError: ErrorsTestSchema.object_from_id(node_id, ctx) must be implemented to load by ID (tried to load from id `2`)`  will raise.

loads option always call `object_from_id`  now.

Reply:
Ahhh, thanks. I was confused because previously, `load_application_object` was called on the _instance_ of an InputObject type. But now, it's called on the _class_. So, to intercept the lookup, you have to define `def self.object_from_id` instead of `def object_from_id`. 

I pushed a change to update the test to reflect this change in behavior.